src/test/pull_hf_model_test.cpp

```diff
@@ -344,3 +345,84 @@ TEST(Libgit2Framework, TimeoutTestProxy) {
 
     git_libgit2_shutdown();
 }
+
+class TestEnabledConfig : public ovms::Config {
+public:
+    TestEnabledConfig() :
+        Config() {
+        std::string port{"9000"};
+        randomizeAndEnsureFree(port);
+        this->serverSettings.grpcPort = std::stoul(port);
+    }
+
+    ovms::ServerSettingsImpl& getServerSettings() {
+        return this->serverSettings;
+    }
+
+    ovms::ModelsSettingsImpl& getModelSettings() {
+        return this->modelsSettings;
+    }
+};
+
+class ServerShutdownGuard {
+    ovms::Server& ovmsServer;
+
+public:
+    ServerShutdownGuard(ovms::Server& ovmsServer) :
+        ovmsServer(ovmsServer) {}
+    ~ServerShutdownGuard() {
+        ovmsServer.shutdownModules();
+    }
+};
+
+TEST(ServerModeTests, ListModelError) {
+    std::unique_ptr<ServerShutdownGuard> serverGuard;
+    ovms::Server& server = ovms::Server::instance();
+    TestEnabledConfig config;
+    config.getServerSettings().serverMode = ovms::LIST_MODELS_MODE;
+    auto retCode = server.startModules(config);
+    EXPECT_TRUE(retCode.ok()) << retCode.string();
+    serverGuard = std::make_unique<ServerShutdownGuard>(server);
+    ASSERT_EQ(server.getModule(ovms::SERVABLES_CONFIG_MANAGER_MODULE_NAME)->getState(), ovms::ModuleState::INITIALIZED);
```

Comment:
Please avoid unit test crashes, we do have too much of those recently
```suggestion
    ASSERT_NE(server.getModule(ovms::SERVABLES_CONFIG_MANAGER_MODULE_NAME), nullptr);
    ASSERT_EQ(server.getModule(ovms::SERVABLES_CONFIG_MANAGER_MODULE_NAME)->getState(), ovms::ModuleState::INITIALIZED);
```